NAMESPACE

```diff
@@ -489,6 +489,7 @@ export(scale_color_fermenter)
 export(scale_color_gradient)
 export(scale_color_gradient2)
 export(scale_color_gradientn)
+export(scale_color_gray)
```

Comment:
Can I get you to remove the two new exports in the NAMESPACE... It is not your fault, but another PR did not get oxygenised and this is now causing your PR to fail check. Removing these lets us handle it in a different PR